mergify_engine/config.py

```diff
@@ -231,11 +232,8 @@ def AccountTokens(v: str) -> typing.Dict[str, str]:
 ORG_ADMIN_GITHUB_APP_OAUTH_TOKEN: github_types.GitHubOAuthToken
 ORG_USER_PERSONAL_TOKEN: github_types.GitHubOAuthToken
 
-configuration_file = os.getenv("MERGIFYENGINE_TEST_SETTINGS")
-if configuration_file:
-    # FIXME: ignore type hit because it's wrong
-    dotenv.load_dotenv(stream=configuration_file, override=True)  # type: ignore
-
+dotenv_path = Path("fake.env")
+dotenv.load_dotenv(dotenv_path=dotenv_path, override=True)
```

Comment:
You cannot break what is existing, i.e., loading the file from the env variable.